cli/js/web/abort_controller.ts

```diff
@@ -3,14 +3,10 @@
 import { AbortSignalImpl, signalAbort } from "./abort_signal.ts";
 
 export class AbortControllerImpl implements AbortController {
-  #signal = new AbortSignalImpl();
-
-  get signal(): AbortSignal {
-    return this.#signal;
-  }
+  readonly signal = new AbortSignalImpl();
```

Comment:
This is wrong.  `readonly` is a TypeScript only constraint.  We need real runtime read-only to be spec compliant.  Please revert.